glide.yaml

```diff
@@ -67,6 +65,42 @@ import:
   version: c7d06af17c68cd34c835053720b21f6549d9b0ee
 - package: github.com/gbraad/go-hvkvp
   version: 0.3
+
+# Get all dependencies for containers/image which is used for OCI image chaching
+- package: github.com/containers/image
+  version: issue-368
+  repo: https://github.com/hferentschik/image.git
```

Comment:
Need TODO here.
```
TODO: Update with upstream repo once pending PRs are merged
```

Reply:
I don't think I need to add this explicitly. I am aware of th